movie_search/models.py

```diff
@@ -18,3 +21,35 @@ class Provider(models.Model):
     def __str__(self):
         """String for representing the Model object."""
         return f"{self.provider_id}: {self.name}"
+
+
+class Video(models.Model):
+   
+    name = models.CharField(max_length=200)
+    key =  models.CharField(max_length=200)
+
+    def __str__(self):
+        """String for representing the Model object."""
+        return f"{self.name}: {self.key}"
+
+class Movie(models.Model):
+    movie_id = models.PositiveSmallIntegerField(default=0)
+    title = models.CharField(max_length=200, null=True, blank=True)
+    backdrop_path = models.CharField(max_length=200, null=True, blank=True)
+    genres = models.ManyToManyField(Genre, related_name="movies", blank=True)
+    tagline = models.CharField(max_length=200, null=True, blank=True)
+    vote_count = models.PositiveSmallIntegerField(default=0)
+    vote_average = models.PositiveSmallIntegerField(default=0)
+    popularity = models.IntegerField(default=0, null=True, blank=True)
+    release_date = models.CharField(max_length=200, null=True, blank=True)
+    runtime = models.IntegerField(default=0, null=True, blank=True)
+    production_company = models.CharField(max_length=200, null=True, blank=True)
+    overview = models.TextField(default="", null=True, blank=True)
+    budget = models.IntegerField(default=0, null=True, blank=True)
+    revenue = models.IntegerField(default=0, null=True, blank=True)
```

Comment:
these are whole numbers, right? no float?

Reply:
> these are whole numbers, right? no float?

They are integers per the movie detail response https://developer.themoviedb.org/reference/movie-details. Is this correct syntax?

Reply:
I see it now there, yes then we're good on these column types.